.travis.yml

```
@@ -26,10 +26,7 @@ script:
   - cargo test && cargo test --no-default-features &&
     cargo test --no-default-features --features std &&
     cargo test --no-default-features --features "std i128" &&
-    cargo test --no-default-features --features "std core_hint_black_box" &&
-    cargo test --no-default-features --features "std const-generics" &&
-    cargo test --no-default-features --features "std i128 core_hint_black_box" &&
-    cargo test --no-default-features --features "std i128 core_hint_black_box const-generics"
+    cargo test --no-default-features --features "std const-generics"
```

I think you also want:

```suggestion
    cargo test --no-default-features --features "std const-generics"
    cargo test --no-default-features --features "std i128 const-generics"
```

---

Yes. Thanks for catching this!